contrib/cmd/seccompagent/config.json

```diff
@@ -0,0 +1,197 @@
+{
+	"ociVersion": "1.0.2-dev",
+	"process": {
+		"terminal": true,
+		"user": {
+			"uid": 0,
+			"gid": 0
+		},
+		"args": [
+			"sh",
+			"-c",
+			"set -x; cd /dev/shm; mkdir test-dir; touch test-file; chmod 777 test-file; ls -l /dev/shm; echo \"Note the agent added a suffix for the directory name and chmod fails\" "
```

Comment:
Perhaps it makes sense to add comments to at least mark those parts of the file that are specific to this demo. This includes this `args` section, and `seccomp` below. Anything else?

Reply:
Or, add a short script instead that would modify config.json to achieve what you need. You can see lots of examples of how to do it with `jq` in `tests/integration`

Reply:
Either way is fine with me -- I just want to be able to find out in 15+ years which parts are specific to the test, and which are whatever `runc spec` generated back in 2021 :)

Reply:
yes, it is a config as generated by runc spec, with only the args and seccomp section modified. I stated that in the README so it is easy for our future selfs :-) (json can't include comments AFAIK).

Reply:
I think he meant that it would be nice if there was a simple script that did:

```sh
% runc spec
% jq '.process.args = [...]' <config.json | sponge config.json
```

Another option would be to put the test commands in a script and execute the script (copying it into the rootfs in the integration test) which would be even cleaner.

But we can always fix that separately (and having an explanation in the readme is probably sufficient).

Reply:
Sure, I can do that in a follow-up PR :)